app/src/main/java/org/sopt/cgv/feature/time/component/ClickableVerticalRegionListInModal.kt

```diff
@@ -0,0 +1,77 @@
+package org.sopt.cgv.feature.time.component
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
+import org.sopt.cgv.core.common.extension.noRippleClickable
+import org.sopt.cgv.core.designsystem.theme.Black
+import org.sopt.cgv.core.designsystem.theme.CGVTheme
+import org.sopt.cgv.core.designsystem.theme.Gray700
+
+@Composable
+fun ClickableVerticalRegionListInModal(
+    list: PersistentList<String>,
+    selectedRegionInModal: MutableState<String>,
```

Comment:
```suggestion
     selectedRegion: String,
    onRegionSelected: (String) -> Unit,
```
State Hoisting을 진행해주신 점 좋습니다!! 다만, 
컴포넌트는 상태 관리가 아닌 UI 표시에만 집중하여 구현해주신다고 보면 될 거 같아요!

Reply:
진짜 이말듣고 이해되었습니다.. 왜 그렇게 해야하는지도 공부했습니다!